src/xlat-i/interfaceBuilder.h

```diff
@@ -347,12 +347,12 @@ namespace Builder {
       : ret(ret_)
       , name(name_)
       , stackSize(stackSize_)
-      , eattrib(0)
+      , attribs(nullptr)
       , sdag(sdag_)
     { }
 
     void addAttribute(BUILDER_ENTRY_ATTRIBUTES attribute) {
-      eattrib |= attribute;
+      attribs = new xi::Attribute(attribute, nullptr, attribs);
```

Comment:
I assume this gets deleted somewhere?

Reply:
Now that you bring my attention to it, this is definitely getting leaked. That being said, deletion would only be necessary at the end of the program's execution and, also because this API is applied in a rather limited capacity, this would probably not be a big deal. I am honestly unsure how memory clean-up is works in xlat-i and, by extension, its API builder. I will have to look into this more and see if `Entry` (the attribute's "owner") is even getting deleted.